src/containers/menus/sidemenu/zoom-controls/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { connect } from 'react-redux';
+
+import * as actions from 'actions/url-actions';
+
+import { loadModules } from 'esri-loader';
+
+import ZoomControlsComponent from './component';
+
+function ZoomControls(props) {
+  const { view, hidden } = props;
+  const [zoomWidget, setZoomWidget] = useState(null);
+
+  // Load custom zoom widget
+  useEffect(() => {
+    const node = document.createElement('div');
+    loadModules(['esri/widgets/Zoom/ZoomViewModel']).then(([ZoomView]) => {
+      const zoomWidget = new ZoomView({
+        view,
+      });
+      setZoomWidget(zoomWidget);
+      if (!hidden) {
+        view.ui.add(node, 'top-right');
+        // eslint-disable-next-line react/jsx-filename-extension
+        ReactDOM.render(<ZoomControlsComponent zoomWidget={zoomWidget} />, node);
+      }
+    }).catch((err) => console.error(err));
```

Comment:
Ah, this is the reason why the zoom disappears sometimes. We could add a task with low priority to check if there is a better solution

Reply:
going to add to JIRA